ios/Classes/BetterPlayerPlugin.m

```diff
@@ -381,13 +381,19 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
         [self onPlayerSetup:player result:result];
     } else {
         NSDictionary* argsMap = call.arguments;
+        id textureIdObject = [argsMap objectForKey:@"textureId"];
+        
+        if (textureIdObject == [NSNull null]) {
```

Comment:
I want to know.
Did you have any error without this condition?

Reply:
Actually, it is the exception in [this PR](https://github.com/dwango-nfc/dwango-app-ch/pull/1787/files), and we all decide to fix it here.